app/kafka/manager/actor/cluster/KafkaStateActor.scala

```diff
@@ -284,7 +625,8 @@ case class OffsetCacheActive(curator: CuratorFramework,
 
   protected def getTopicDescription(topic: String, interactive: Boolean) : Option[TopicDescription] = topicDescriptions(topic, interactive)
   
-  def start():  Unit = {
+  override def start():  Unit = {
+    super.start()
```

Comment:
wouldn't this start Kafka managed offsets cache and let it poll offsets periodically even when consumer polling option is disabled?